### PR TITLE
added an important blank line

### PR DESCRIPTION
**Pull request by toddalbert**
We really needed this extra line. You're welcome. I know, I deserve a raise.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const express = require('express')
 const bodyParser = require('body-parser')
+
```
Comment (toddalbert) [inline app.js:3]: This is the best code I've ever seen you write! Wow. Blown away. Nice job.

Comment (toddalbert) [reply app.js:3]: Thanks, man. Really worked hard on this one.

Comment (bobby-langley-personal) [inline app.js:3]: This will do.

**Review by RebeccaBakels — APPROVED** [review]
Garbage.

**Pull request merged**